setup.py

```diff
@@ -46,7 +47,7 @@
     tests_require=tests_require,
     install_requires=[
         'openeo>=0.13.0a2.*',
-        'openeo_driver>=0.27.0.dev',
+        'openeo_driver>=0.28.0a1.dev',
```

Comment:
FYI: this dep has been bumped to 0.28.3a1 in master already (1d72b14e1fd9e87213526b04559c8c775c003ccf)

Reply:
I just noticed that upgrading to openeo_driver 0.28.0a1.* brakes a whole lot of tests, 

for example:
```
ERROR tests/test_views.py::TestBatchJobs::test_cancel_job[0.4.0] - AttributeError: type object 'ElasticJobRegistry' has no attribute '_log'
ERROR tests/test_views.py::TestBatchJobs::test_cancel_job[1.0.0] - AttributeError: type object 'ElasticJobRegistry' has no attribute '_log'
```

Reply:
Have merged master into the issues's feature branch. Should be fixed now.